project-chip/connectedhomeip
# [Tizen] [lighting-app] Add GPIO control and dbus UI

## Description
Add GPIO support for controlling an LED in the Tizen lighting app example. Also, add support to controlling the lighting app using the dbus interface. 

## Testing
1. Build using: 
``` sh
./scripts/build/build_examples.py --target tizen-arm-light-with-ui --enable-flashbundle build
```
2. Check onOff value
``` sh
gdbus call --system --dest org.tizen.matter.example.lighting --object-path /app  --method org.freedesktop.DBus.Properties.GetAll org.tizen.matter.example.lighting.OnOff
```
4. Set onOff value
``` sh
gdbus call --system --dest org.tizen.matter.example.lighting --object-path /app  --method org.freedesktop.DBus.Properties.Set org.tizen.matter.example.lighting.OnOff onOff "<true>"
```
5. Use chip-tool to commission the device
6. Use chip-tool to change onOff value
``` sh 
./out/linux-x64-chip-tool/chip-tool onoff off 1 1
```
7. Check onOff value as at point 2
